readme.md

```
@@ -94,31 +363,241 @@
+
+### Base URL
+```
+Production: https://api.ipfs-gateway.com
+Development: http://localhost:5000
+```
+
+All examples below are shown as absolute paths. In the current backend implementation, endpoints are mounted under `/api/v1/users`.
+
+### Authentication
+All API requests (except registration) require an API key in the header:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)